src/main/resources/plugin.yml

```diff
@@ -11,6 +13,9 @@ commands:
   color:
     description: Sets your chat or name color
     usage: /color <chat|name> [color]
+  hcadvancement:
+    description: Manages Hack Club advancements
+    usage: /hcadvancement [grant|revoke] [player] [advancement]
```

Comment:
Does it even need its own command?
this api provides its own command as well
and there is a vanilla advancement command that this (should) work with (does this api break it that much that it doesnt? is that why it provides its own?)

Reply:
Oh, I didn't even realize that the api already provided its own command. I'll revert that commit.
The vanilla advancement command doesn't work because this doesn't follow the vanilla advancement api using a datapack, like the old API did. It stores and processes all of the advancement data itself in a sqlite database (although that can be changed). So yeah its not great that it doesnt work with the built in API but it does allow for a lot more customization and flexibility.